platform/error/platform-error-impl/src/main/resources/error.html

```diff
@@ -0,0 +1,128 @@
+<!--
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+-->
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<head>
+  <title>Error</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <style>
+    * {
+      -webkit-box-sizing: border-box;
+      -moz-box-sizing: border-box;
+      box-sizing: border-box;
+    }
+
+    body {
+      font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif;
+      font-size: 15px;
+      line-height: 1.42857143;
+      color: #2c3e50;
+      background-color: #ffffff;
+      margin: 0;
+    }
+
+    @media (min-width: 768px) {
+      .container {
+        width: 750px;
+      }
+    }
+
+    @media (min-width: 992px) {
+      .container {
+        width: 970px;
+      }
+    }
+
+    @media (min-width: 1200px) {
+      .container {
+        width: 1170px;
+      }
+    }
+
+    .container {
+      margin-right: auto;
+      margin-left: auto;
+      padding-left: 15px;
+      padding-right: 15px;
+    }
+
+    .page-header {
+      padding-bottom: 9px;
+      margin: 42px 0 21px;
+      border-bottom: 1px solid transparent;
+    }
+
+    article, aside, details, figcaption, figure, footer, header, main, nav, section, summary {
+      display: block;
+    }
+
+    ul, ol {
+      margin-top: 0;
+      margin-bottom: 10px;
+    }
+
+    .full-height {
+      height: 100%;
+    }
+
+    a {
+      color: #18bc9c;
+      text-decoration: none;
+    }
+
+    a {
+      background: transparent;
+    }
+
+    .error-code {
```

Comment:
📓 I simplified and modified the `.error-code` and `.error-message` styles in order to get the two text blocks center-aligned vertical. Please confirm this is precise enough.